helpers/request.ts

```diff
@@ -310,3 +310,25 @@ export function requestTryAndRepeatUntil<T>(
     }
   );
 }
+
+export function isSuccessStatus(status: number) {
+  return status >= HttpStatus.ok && status < HttpStatus.multipleChoices;
```

Comment:
I would keep the magic numbers here, because a success status = 2xx, not really that the status is between `ok` and `multipleChoices` (wat it mean ?? )

Reply:
Yeah indeed